### PR TITLE
[TERM] : Added new term entry in python for ljust method in strings

### LINKED ISSUE #3284: [Term Entry] Python Strings .ljust()
### Reason/inspiration (optional)

We would like a new term entry on the `.ljust()` method for Python: [Strings](https://www.codecademy.com/resources/docs/python/strings). The entry should go in a new file under `docs/content/python/concepts/strings/terms/ljust/ljust.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```py `code snippet.
- A “Codebyte Example” section with a ` ```codebyte/python ` snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Atharva1723**
Resolves #3284

### Description
Added a new term entry for the ljust() method. The description provides an explanation of the method, detailing its functionality in left-aligning a string with a specified fill character (defaulting to space).

### Type of Change

Documentation Update

- Adding a new entry
- Updating the documentation

### Checklist

- [ ✔] All writings are my own.
- [ ✔ ] My entry follows the Codecademy Docs style guide.
- [ ✔ ] My changes generate no new warnings.
- [ ✔] I have performed a self-review of my own writing and code.
- [ ✔] I have checked my entry and corrected any misspellings.
- [ ✔ ] I have made corresponding changes to the documentation if needed.
- [ ✔ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ✔ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ✔] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.ljust()`** method left-aligns a string with a specified fill character
+
+## Syntax
+
+```pseudo
+string.ljust(length,fillchar)
```
Comment (ishg-153) [inline content/python/concepts/strings/terms/ljust/ljust.md:21]:
```suggestion
string.ljust(length, fillchar)
```

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+The `.ljust()` method is used to left align the string with the width specified. If the specified width is more than the length of the string, then the remaining part of the string is filled with `fillchar`
```
Comment (ishg-153) [inline content/python/concepts/strings/terms/ljust/ljust.md:24]:
```suggestion
- `length`:
- `fillchar`:
```
Explain both these variables that were used in syntax.

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Codebyte Example
+
+The following code is runnable and uses the `.ljust()` method.
+
+```codebyte/python
+exampleString = 'User, Welcome to docs'
+exampleString = exampleString.ljust(22, '!')
+print(exampleString)
+
```
Comment (ishg-153) [inline content/python/concepts/strings/terms/ljust/ljust.md:54]:
```suggestion
exampleString = 'User, Welcome to docs'
exampleString = exampleString.ljust(22, '!')
print(exampleString)
```
Removed a blank line.

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+left_aligned_text = text.ljust(20, '!')
+print(left_aligned_text)
+
+```
+
+This code will result in the following output:
+
+```shell
+Hello Contributors!!
+
```
Comment (ishg-153) [inline content/python/concepts/strings/terms/ljust/ljust.md:41]:
```suggestion
Hello Contributors!!
```
Removed a blank line.

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+## Examples
+
+The default fillchar is a blank space. The original string is retrieved if the width is less than or equal to the given string length.
+
+```python
+text = 'Hello Contributors'
+left_aligned_text = text.ljust(20, '!')
+print(left_aligned_text)
+
+```
```
Comment (ishg-153) [inline content/python/concepts/strings/terms/ljust/ljust.md:35]:
```suggestion
```python
text = 'Hello Contributors'
left_aligned_text = text.ljust(20, '!')
print(left_aligned_text)
```
```
Removed a blank line.

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+```pseudo
+string.ljust(length,fillchar)
+```
+
+The `.ljust()` method is used to left align the string with the width specified. If the specified width is more than the length of the string, then the remaining part of the string is filled with `fillchar`
+
+## Examples
+
+The default fillchar is a blank space. The original string is retrieved if the width is less than or equal to the given string length.
```
Comment (ishg-153) [inline content/python/concepts/strings/terms/ljust/ljust.md:28]:
```suggestion
The default `fillchar` is a blank space. The original string is retrieved if the length is less than or equal to the given string's length.
```
Also, move this above the "Example" header.

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.ljust()`** method left-aligns a string with a specified fill character
```
Comment (ishg-153) [inline content/python/concepts/strings/terms/ljust/ljust.md:16]: Please add the following points:
- is a method for strings
- returns a left0aligned string of the length given
Feel free to cover other relevant information.

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+This code will result in the following output:
+
+```shell
+Hello Contributors!!
+
+```
+
+> **Note:** `.ljust()` returns a new string and does not modify the original string and only one specific character can be mentioned to fill the remaining part of the string with fillchar.
```
Comment (ishg-153) [inline content/python/concepts/strings/terms/ljust/ljust.md:44]:
```suggestion
> **Note:** `.ljust()` returns a new string and does not modify the original string. Only one specific character can be mentioned to fill the remaining part of the string with fillchar.
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
>Make sure to read all the comments and suggestions carefully.

@Atharva1723 I have added my comments; please address them.

**Comment by Atharva1723** [comment]
@ishg-153 I have commited your suggestions and also added some from my side

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The method **`.ljust()`** adds padding of a specified length to the right of a given string. The user can specify a character to use as the padding. The number of padding characters added is equal to the length specified in the method call minus the length of the string provided.
```
Comment (ishg-153) [inline content/python/concepts/strings/terms/ljust/ljust.md:16]:
```suggestion
The method **`.ljust()`** left aligns a string for the given length of characters. If the length of the string is less than the parameter given, then padding is added to the right of a given string. The user can specify a character to use as the padding. The number of padding characters added is the length of the string provided subtracted from the length specified in the method call.
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Atharva1723 I have suggested one change. Please make that. Everything else looks good.

**Review by ishg-153 — APPROVED** [review]
@Atharva1723 This looks ready for a second review.

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+print(left_aligned_text)
+```
+
+This code will result in the following output:
+
+```shell
+Hello Contributors!!
+```
+
+> **Note:** `.ljust()` returns a new string and does not modify the original string. Only one specific character can be mentioned to fill the remaining part of the string with fillchar.
```
Comment (avdhoottt) [inline content/python/concepts/strings/terms/ljust/ljust.md:49]:
```suggestion
> **Note:** `.ljust()` returns a new string and does not modify the original string. Only one specific character can be mentioned to fill the remaining part of the string with the parameter `fillchar`.
```
Some readers might get confused if they directly read the sentence. I think it'll be a good practice to mention that it is a parameter.

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+- `string`: The string to be modified with padding.
+- `length`: An integer used to determine how many characters of padding will be added. The number of padding characters will be equal to `length` minus the length of `string`.
+
+### Optional Parameters
+
+- `fillchar`: A character to fill the missing space (to the right of the string). Default is " " (space).
+
+## Examples
+
+The default `fillchar` is a blank space. The original string is retrieved if the length is less than or equal to the given string's length.
```
Comment (avdhoottt) [inline content/python/concepts/strings/terms/ljust/ljust.md:35]:
```suggestion
The default value for the parameter `fillchar` is a blank space. The original string is retrieved if the length is less than or equal to the given string's length.
```
Same suggestion for this, mentioning that it is a parameter will be helpful to the reader.

File: content/python/concepts/strings/terms/ljust/ljust.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+### Required Parameters
+
+- `string`: The string to be modified with padding.
+- `length`: An integer used to determine how many characters of padding will be added. The number of padding characters will be equal to `length` minus the length of `string`.
+
+### Optional Parameters
+
+- `fillchar`: A character to fill the missing space (to the right of the string). Default is " " (space).
```
Comment (avdhoottt) [inline content/python/concepts/strings/terms/ljust/ljust.md:31]:
```suggestion
- `fillchar`: A character to fill the missing space (to the right of the string). The default value is " " (space).
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Nice Work @Atharva1723, I have suggested some minor changes, make them and I'll merge it.

**Review by avdhoottt — APPROVED** [review]
Ready to get merged!

**Pull request merged**